src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1227,15 +1227,15 @@ private function addExceptionsSection(ArrayNodeDefinition $rootNode)
                             ->scalarNode('log_level')
                                 ->info('The level of log message. Null to let Symfony decide.')
                                 ->validate()
-                                    ->ifTrue(function ($v) use ($logLevels) { return !\in_array($v, $logLevels); })
+                                    ->ifTrue(function ($v) use ($logLevels) { return null !== $v && !\in_array($v, $logLevels, true); })
                                     ->thenInvalid(sprintf('The log level is not valid. Pick one among "%s".', implode('", "', $logLevels)))
                                 ->end()
                                 ->defaultNull()
                             ->end()
                             ->scalarNode('status_code')
                                 ->info('The status code of the response. Null to let Symfony decide.')
                                 ->validate()
-                                    ->ifTrue(function ($v) { return $v < 100 || $v > 599; })
+                                    ->ifTrue(function ($v) { return null !== $v && ($v < 100 || $v > 599); })
```

Comment:
Looking at the code in ErrorListener, we can also use `0` instead of `null`.
Let's fix the info message and allow it instead? That'd fix the XSD.

Reply:
Done.
Allowed to specify both `null` and `0` to cover XML cases (only `0` is possible to be used) and allow in yaml use preferred format by developers.